MyCryptoHQ/MyCrypto
# Feature/unstoppabledomains

## Description
Integration with Unstoppabledomains using @unstoppabledomains/resolution for resolving ens domain names as well as .zil and .crypto.


## Changes
Added UnstoppableResolution service that handles the domain resolution and error messages

## Steps to Test

* Go to send Assets page
* Choose assets to send (eth as an example)
* Type in recipient field any .eth .luxe .zil or .crypto domains
* press tab
* see the resolved domain's address if any or a corresponding error such as Domain is not registered

## Quality Assurance

- [x] If code is copied from existing directories, there is an explanation of why this is necessary in the description/changes, and all copying is done in separate commits to make them easy to filter out
